pkg/queue/breaker.go

```diff
@@ -67,6 +67,15 @@ func NewBreaker(params BreakerParams) *Breaker {
 	}
 }
 
+// Returns if the breaker can accept a request.
+func (b *Breaker) HasCapacity() bool {
```

Comment:
Fix the linting comment please.

Reply:
hrrr, the function name in the beginning :|